cpp/include/raft/matrix/detail/select_warpsort.cuh

```diff
@@ -324,7 +322,7 @@ class warp_sort_filtered : public warp_sort<Capacity, Ascending, T, IdxT> {
 
   __device__ __forceinline__ void merge_buf_()
   {
-    topk::bitonic<kMaxBufLen>(!Ascending, kWarpWidth).sort(val_buf_, idx_buf_);
+    util::bitonic<kMaxBufLen>(!Ascending, kWarpWidth).sort(val_buf_, idx_buf_);
```

Comment:
There's a lot of explicit `__device__ __forceinline__` across these files. Can you do a search and replace and use the `_RAFT_DEVICE _RAFT_FORCEINLINE` from `macros.hpp`?